ansible/roles/dns/vars/main.yml

```diff
@@ -1,3 +1,5 @@
 bind_zones:
   conference.fosdem.net:
```

Comment:
I've pushed this already and it appears to be working; the ansible module we're using generates named.conf.local